anitya/tests/lib/backends/test_gnome.py

```diff
@@ -27,13 +27,10 @@
 import sys
 import os
 
-sys.path.insert(0, os.path.join(os.path.dirname(
-    os.path.abspath(__file__)), '..'))
-
```

Comment:
sys and os are still imported here while you dropped them from other tests, intended?